infra_event_notifier/backends/datadog.py

```diff
@@ -0,0 +1,44 @@
+import time
+import json
+
+import urllib.request
+
+# from urllib.request import Request, urlopen
+from typing import Mapping
+
+
+def notify_datadog(
+    title: str,
+    text: str,
+    tags: Mapping[str, str],
+    datadog_api_key: str,
+    alert_type: str = "user_update",
+    epoch: int = None,
```

Comment:
default doesn't match the type annotation, i'd suggest running a typechecker over this as this line should fail

Reply:
Please remove this attribute. There is no need to allow the callsite to specify the timestamp of the notification. You can rely on the system timestamp.